alerting/src/test/kotlin/org/opensearch/alerting/DocumentMonitorRunnerIT.kt

```diff
@@ -329,8 +331,16 @@ class DocumentMonitorRunnerIT : AlertingRestTestCase() {
 
         indexDoc(testIndex, "1", testDoc)
         indexDoc(testIndex2, "5", testDoc)
+        executeMonitor(monitor.id)
+
+        // clear previous findings and alerts
+        deleteIndex(ALL_FINDING_INDEX_PATTERN)
+        deleteIndex(ALL_ALERT_INDEX_PATTERN)
```

Comment:
Should we add asserts for the first run (``executeMonitor``)  too, to ensure documents in ``testIndex1`` and ``testIndex2`` are both already monitored.

Reply:
done